core/variant/variant.cpp

```diff
@@ -1801,6 +1834,9 @@ String Variant::stringify(int recursion_count) const {
 		case PACKED_COLOR_ARRAY: {
 			return stringify_vector(operator PackedColorArray(), recursion_count);
 		}
+		case PACKED_VECTOR4_ARRAY: {
+			return stringify_vector(operator Vector<Vector4>(), recursion_count);
```

Comment:
```suggestion
			return stringify_vector(operator PackedVector4Array(), recursion_count);
```
Following previous cleanup

Reply:
Let's just get these for uniformity